tests_notebooks/test_notification.py

```diff
@@ -0,0 +1,37 @@
+import time
+from pathlib import Path
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webdriver import WebDriver
+
+
+def test_home_notification(selenium_driver, final_screenshot):
+    config_dir = Path.home() / ".aiidalab"
+    warning_file = config_dir / "home_app_warning.md"
+
+    # If the file exists, we are testing locally
+    # We should make sure not to overwrite or remove the file
+    testing_locally = warning_file.exists()
```

Comment:
I think this case would be better handled as a fixture. The test should not even touch / see the normal config dir. Instead you could e.g. have the config dir specified as a global variable, that would get overridden in the fixture with `tmp_path / ".aiidalab"`.

Reply:
Not sure what you mean, but also, maybe you can help me first figure out why the test is not working as is? 😅🙏